Vagrantfile

```diff
@@ -36,10 +36,13 @@ Vagrant.configure("2") do |config|
     sudo apt-get install xubuntu-desktop gksu leafpad synaptic
     sudo add-apt-repository -y "ppa:webupd8team/sublime-text-3"
     sudo apt-add-repository -y "ppa:webupd8team/atom"
+    curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+    sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+    sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
```

Comment:
Could we also do `sudo echo "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list` without the `sh -c`?